### PR TITLE
build(docker): add dockerize image

### PR DESCRIPTION
**Pull request by alekseyolg**
The Dockerezer image currently on the Helm Chart is too old and has critical vulnerabilities, for example:

`
busybox, 1.26.2-r5, CVE-2017-16544 (https://nvd.nist.gov/vuln/detail/CVE-2017-16544)
`

This pr is designed to solve this problem

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [x] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by alekseyolg** [comment]
Hi all!
I can not understand the reason for the error when running the test:
```
Python Misc / pre-commit (3.9) (pull_request) 
```
Maybe someone can advise?

File: dockerize.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM alpine:latest
+
+ARG DOCKERIZE_VERSION=v0.6.1
```
Comment (craig-rueda) [inline dockerize.Dockerfile:3]: Do we need this? I think it's preferable to just switch over to the "official" image

Comment (alekseyolg) [reply dockerize.Dockerfile:3]: @craig-rueda I don't understand what you mean by "official". I found only this repository on dockerhub (on the attached image). The youngest image from there is 4 years old. If I did not find a link to some official image, then I would be grateful if you show it to me, and I, in turn, could study this issue.

About the busybox image: dockereze requires openssl to work, and busybox  doesn't have a package manager to install it, so in my opinion the best way is to use the alpine base image.
image

**Review by craig-rueda — APPROVED** [review]
Looks like CI is failing, otherwise LGTM

**Comment by alekseyolg** [comment]
@craig-rueda I managed to solve the problem - now all processes work correctly!

File: dockerize.Dockerfile (hunk 1 above)
Comment (alekseyolg) [reply dockerize.Dockerfile]: We need this image in order to expect connection to postgreSQL and Redis. If you do not use this image, then users may make an error when starting the chart and will endlessly receive a crashloopbackoff, and this image in the work logs clearly shows that the connection failed and repeats attempts to contact the destination (in my opinion, this is a good idea ).

**Pull request merged**